src/index.html

```diff
@@ -11,6 +11,30 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-box"
+      action="/search_page.php"
+      method="post"
+      data-qa="big"
+    >
+      <input
+        class="search-input search-input--big "
+        type="search"
```

Comment:
```suggestion
      <input
        class="search-input"
        type="search"
```
У нас немає ніякого проміжного input, відповідно, чому має бути великий і малий input? У нас є input та input--small (--small - модифікатор). Тому всі стилі які прописані для “великого” інпуту ми задаємо для всіх інпутів в цілому, а за допомогою модифіктора small міняємо тільки потрібні значення, щоб наш інпут став малим.

Reply:
Дякую! Змінив роботу шрифтів і виправив назви классів відповідно до зауважень